CONTRIBUTING.md

```diff
@@ -23,6 +23,94 @@ If you, as a community contributor, wish to work on more extensive features, ple
+    "@medusajs/sales-channel": "file:../medusa/packages/modules/sales-channel",
+    "@medusajs/stock-location": "file:../medusa/packages/modules/stock-location",
+    "@medusajs/store": "file:../medusa/packages/modules/store",
+    "@medusajs/tax": "file:../medusa/packages/modules/tax",
+    "@medusajs/user": "file:../medusa/packages/modules/user",
+    "@medusajs/workflow-engine-inmemory": "file:../medusa/packages/modules/workflow-engine-inmemory",
+    "@medusajs/link-modules": "file:../medusa/packages/modules/link-modules",
+    "@medusajs/admin-bundler": "file:../medusa/packages/admin/admin-bundler",
+    "@medusajs/admin-sdk": "file:../medusa/packages/admin/admin-sdk",
+    "@medusajs/admin-shared": "file:../medusa/packages/admin/admin-shared",
+    "@medusajs/dashboard": "file:../medusa/packages/admin/dashboard",
+    "@medusajs/admin-vite-plugin": "file:../medusa/packages/admin/admin-vite-plugin",
+    "@medusajs/ui": "file:../medusa/packages/design-system/ui",
+    "@medusajs/icons": "file:../medusa/packages/design-system/icons",
+    "@medusajs/toolbox": "file:../medusa/packages/design-system/toolbox",
+    "@medusajs/ui-preset": "file:../medusa/packages/design-system/ui-preset",
+    "@medusajs/utils": "file:../medusa/packages/core/utils",
+    "@medusajs/types": "file:../medusa/packages/core/types",
+    "@medusajs/core-flows": "file:../medusa/packages/core/core-flows",
+    "@medusajs/orchestration": "file:../medusa/packages/core/orchestration",
+    "@medusajs/cli": "file:../medusa/packages/cli/medusa-cli",
+    "@medusajs/modules-sdk": "file:../medusa/packages/core/modules-sdk",
+    "@medusajs/workflows-sdk": "file:../medusa/packages/core/workflows-sdk",
+    "@medusajs/framework": "file:../medusa/packages/core/framework",
+    "@medusajs/auth-emailpass": "file:../medusa/packages/modules/providers/auth-emailpass",
+    "@medusajs/locking-redis": "file:../medusa/packages/modules/providers/locking-redis",
+    "@medusajs/locking-postgres": "file:../medusa/packages/modules/providers/locking-postgres",
+    "@medusajs/telemetry": "file:../medusa/packages/medusa-telemetry",
+    "@medusajs/settings": "file:../medusa/packages/modules/settings",
+    "@medusajs/draft-order": "file:../medusa/packages/plugins/draft-order",
+    "@medusajs/deps": "file:../medusa/packages/deps",
+    "@medusajs/caching-redis": "file:../medusa/packages/modules/providers/caching-redis",
+    "@medusajs/caching": "file:../medusa/packages/modules/caching"
+}
+```
+
+4. Each time you make a change in the forked Medusa repository, you need to build the packages where the modifications took place with `yarn build`. Some packages have a watch script, so you can execute `yarn watch` once and it will automatically build on changes. 
+
+5. To have this changes in your test project, you need to navigate to it and run `rm -R node_modules && yarn && yarn dev` so that node_modules is regenerated with the newly built contents from the previous step. 
```

Comment:
~~~suggestion
3. After building changes in the forked medusa repository, run the following command in the test project to regenerate the `node_modules` directory with the newly built contents from the previous step:

```
rm -R node_modules && yarn && yarn dev
```
~~~